### DISCUSSION TITLE
Can't get it started

### DISCUSSION THREAD

**Discussion by Capcave** [syssi/homeassistant-goecharger-mqtt#75] [Q&A]
I can't seem to get this to work. I have installed a MQTT broker which seems to be running. On the app I have enabled the MQTT API (without a username/password) and pointed it at the IP number of my HomeAssistant server. RE-installed this with the correct serial number, restarted and no stats are coming through. 
Any ideas? 

Thanks

**Comment by syssi** [comment]
Please follow this click path at the Home Assistant UI:

1. Settings
2. Devices
3. Click "configure" on the "MQTT" integration tile
4. Go to the "Listen to a topic" section
5. Enter "#" as topic
6. Start listening

Do you see traffic (messages)?

**Reply by Capcave** [reply]
That was quick. Yes - thats got messages coming through from the charger

**Reply by syssi** [reply]
Perfect! Could you make sure you didn't hit this issue: https://github.com/syssi/homeassistant-goecharger-mqtt/issues/67

Please rename the base topic as workaround.

**Reply by Capcave** [reply]
Tried with and without a leading slash and no data is being shown

**Reply by syssi** [reply]
Could you provide some examples messages received using the "Listen to a topic" guide? I would like make sure your device uses protocol version 2.

**Comment by Capcave** [comment]
Here you go (Serial and Wifi hidden)
`{"version":"B","tme":"2602231632","rbc":"68","rbt":"8320884","car":"4","amp":"16","err":"0","ast":"0","alw":"0","stp":"0","cbl":"32","pha":"56","tmp":"8","tma":[6.50,5.50,5.50,5.38,-0.13,-0.13],"amt":"32","dws":"607321","dwo":"0","adi":"0","uby":"0","eto":"18140","wst":"3","txi":"2","nrg":[225,222,224,2,0,0,0,0,0,0,0,0,0,0,0,0],"fwv":"041.0","sse":"xxxxx","wss":"xxxxx","wke":"********","wen":"1","cdi":"0","tof":"101","tds":"1","lbr":"255","aho":"3","afi":"7","azo":"0","ama":"32","al1":"10","al2":"16","al3":"20","al4":"24","al5":"32","cid":"255","cch":"65535","cfi":"65280","lse":"0","ust":"0","wak":"","r1x":"2","dto":"0","nmo":"0","sch":"AAAAAAAAAAArKOgB","sdp":"0","eca":"0","ecr":"0","ecd":"0","ec4":"0","ec5":"0","ec6":"0","ec7":"0","ec8":"0","ec9":"0","ec1":"0","rca":"","rcr":"","rcd":"","rc4":"","rc5":"","rc6":"","rc7":"","rc8":"","rc9":"","rc1":"","rna":"","rnm":"","rne":"","rn4":"","rn5":"","rn6":"","rn7":"","rn8":"","rn9":"","rn1":"","loe":0,"lot":0,"lom":0,"lop":0,"log":"","lon":0,"lof":0,"loa":0,"lch":3943}                      `

**Reply by syssi** [reply]
This is a API version 1 response. Your charger isn't supported.

**Reply by syssi** [reply]
Please subscribe this issue: https://github.com/syssi/homeassistant-goecharger-mqtt/issues/22

I hope I can add protocol v1 support some day.

**Reply by Capcave** [reply]
Aha ok - that would explain it. thanks for your help.
